### ISSUE TITLE
Add "Kubernetes, the Pulumi way" landing page and supporting docs

### ISSUE THREAD

**Issue by hausdorff** [pulumi/pulumi-kubernetes#91]
This page should clearly explain the Pulumi value proposition as it relates directly to Kubernetes, as well as how to get started. The messaging should be targeted to appeal directly to the audience running Kubernetes.

cc @joeduffy @lukehoban @mehzer

**Comment by hausdorff** [comment]
We have decided to adopt a fairly staid style for our docs, so we are going to close this out.

**Comment by mehzer** [comment]
What does this even mean?

**Comment by mehzer** [comment]
I don't understand why this is being closed, so I'm reopening. 
What is the documentation plan on IO for Kubernetes? cc @lukehoban

**Comment by hausdorff** [comment]
I just mean there will not be a marketing "pitch" page in the .io site -- the scope of the docs is on how to make users successful when they use Pulumi, rather than to pitch them that they _should_.

This does not obviate the need for a nice pitch in other places (_e.g._, .com), of course -- it only reflects an acceptance of the belief that we need to change our docs to be more focused on this specific goal in order to be successful here.

Happy to hear other opinions about this. This comes from a conversation I had with @lukehoban -- feel free to fill in any blanks/correct things if I misunderstood.

**Comment by joeduffy** [comment]
What do you think about a cloud-specific "Why Pulumi?" page? Kind of like https://pulumi.io/reference/vs/index.html, but describes -- in technical terms -- why you would choose Pulumi for Kubernetes, AWS, GCP, ...

@hausdorff, you have (rightfully) said before that landing on pulumi.io is under-whelming for Kubernetes, even if you land on code (which currently you do not by default). Although I agree our technical documentation shouldn't be overly marketing in its style, we do want to tell the story of Pulumi and why someone might be interested in using it (concrete technical advantages).

**Comment by hausdorff** [comment]
@joeduffy @lukehoban I'm not opposed to this approach but since we've gone through ~3 iterations of what a solution to this issue should look like, I'd like to propose that we sit down and make sure the plan of record is at least directionally approaching the final plan for this issue. :)

**Comment by hausdorff** [comment]
Ok, per separate conversations with @joeduffy and @lukehoban we're going to push this out and concentrate on the Kubernetes the Prod Way workstream.

**Comment by lukehoban** [comment]
We're going to move the existing docs into the https://github.com/pulumi/kubernetes-the-prod-way repo where we can iterate on them.  I've opened https://github.com/pulumi/kubernetes-the-prod-way/issues/14.

We'll add additional docs site content and/or www.pulumi.com content in the near future once this is more fully fleshed out.

Closing this out for now to track concrete near term plan in the issue above.
